hazelcast/src/test/java/com/hazelcast/sql/impl/exec/scan/index/IndexEqualsFilterIterationTest.java

```diff
@@ -54,27 +54,27 @@ private void checkIteratorSimple(IndexType indexType) {
         ExpressionEvalContext evalContext = SimpleExpressionEvalContext.create();
 
         // Check missing value.
-        checkIterator(new IndexEqualsFilter(intValue(1)).getEntries(index, evalContext));
+        checkIterator(new IndexEqualsFilter(intValue(1)).getEntries(index, false, evalContext));
```

Comment:
Perhaps we should add unit tests for the reverse direction in `IndexEqualsFilterIterationTest`, `IndexInFilterIterationTest` and `IndexRangeFilterIterationTest`. These are the isolated tests for iteration over entries for the given filter.

Reply:
Added tests in the https://github.com/hazelcast/hazelcast/pull/18013/commits/4b9dd4ebbd0c7bc1bf3665371d8a98ac55cd3ce7